docs/pangeo-showcase.rst

```diff
@@ -26,41 +26,47 @@ Fall 2021 Showcase Recorded and Upcoming Presentations
-     - `Scalable Sustainability with the Planetary Computer <https://discourse.pangeo.io/t/october-20-2021-scalable-sustainability-with-the-planetary-computer/1874>`_ |Augspurger DOI Badge|
-   * - 2021-10-27 4PM EDT
-     - Kata Martin, Joe Hamman, and Jeremy Freeman, CarbonPlan
-     - `A New Toolkit for Visualizing Zarr Data in Web Maps <https://discourse.pangeo.io/t/october-27-2021-a-new-toolkit-for-visualizing-zarr-data-in-web-maps/1891>`_ |Martin DOI Badge|
-   * - 2021-11-03 12PM EDT
-     - Gunnar Brandt and  Norman Fomferra, Brockmann Consult GmbH
-     - `xcube - Python package for Earth Observation data cubes <https://discourse.pangeo.io/t/november-3-2021-xcube-python-package-for-earth-observation-data-cubes/1903>`_ |Brandt DOI Badge|
-   * - 2021-11-10 4PM EDT
-     - John Clyne, NCAR
-     - `Introducing Project Raijin, Community Geoscience Analysis Tools for Unstructured Mesh Data <https://discourse.pangeo.io/t/november-10-2021-introducing-project-raijin-community-geoscience-analysis-tools-for-unstructured-mesh-data/1923>`_ |Clyne DOI Badge|
-   * - 2021-11-17 12PM EDT
-     - Deepak Cherian, NCAR
-     - `flox: Fast & furious GroupBy reductions with Dask at Pangeo-scale <https://discourse.pangeo.io/t/november-17-2021-flox-fast-furious-groupby-reductions-with-dask-at-pangeo-scale/2016>`_ |Cherian DOI Badge|
+   * - Date
+     - Speaker
+     - Title
+   * - 2022-11-16 12PM EST
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-11-09 4PM EST
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-11-02 12PM EDT
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-10-26 4PM EDT
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-10-19 12PM EDT
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-10-12 4PM EDT
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-10-05 12PM EDT
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-09-28 4PM EDT
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-09-21 12PM EDT
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-09-21 12PM EDT
+     - TBD
+     - `TBD <https://discourse.pangeo.io/>`_ |name DOI Badge|
+   * - 2022-09-14 4PM EDT
+     - Peter Marsh
+     - `Accessing NetCDF and GRIB file collections as cloud-native virtual datasets using Kerchunk <https://discourse.pangeo.io/>`_ |name DOI Badge|
      
 Please note that the Pangeo Showcase will go on Winter Break after the talk on Nov. 17th. We will resume Showcase presentations in January 2022. If you are intersted in presenting this Spring, please makes sure to `fill out this short form <https://forms.gle/QwxKusVvrvDakSNs8>`_.
```

Comment:
Should be January 2023, not January 2022.   
And please change the following sentence to: "If you are interested in presenting in 2023, "